### PR TITLE
Setup prettier in all examples

### PR DESCRIPTION
**Pull request by sonalivdeshpande**

### CODE REVIEW

**Comment by Josmithr** [comment]
I would recommend using our baseline eslint and prettier configs from fluid-framework if possible.

- eslint: https://github.com/microsoft/FluidFramework/blob/main/common/build/eslint-config-fluid/recommended.js
- prettier: https://github.com/microsoft/FluidFramework/blob/main/common/build/build-common/prettier.config.cjs

**Comment by Josmithr** [comment]
I would also recommend adding prettier as the default formatter in the vscode settings file, and adding "format-on-save". That way developer will get the desired formatting without having to run the script.

```json
"editor.defaultFormatter": "esbenp.prettier-vscode",
"editor.formatOnSave": true,
```

File: angular-demo/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# Dependencies
+node_modules
+
+# Build output
+dist
+build
+
+# VSCode settings
+.vscode
```
Comment (Josmithr) [inline angular-demo/.prettierignore:12]: Do we need to ignore the vscode settings? Prettier does a pretty good job formatting json.

**Comment by Josmithr** [comment]
Is format on save working correctly in vscode with the current setup? Just wondering if we need to add a dependency on prettier from the root package.json + add the format-on-save config to the repo root vscode settings.

**Review by Josmithr — APPROVED** [review]
I would suggest extending the base prettier config from build-common in these packages as well. Otherwise, looks good to me!

**Review by Josmithr — APPROVED** [review]

**Review by Josmithr — APPROVED** [review]

**Comment by sonalivdeshpande** [comment]
> Is format on save working correctly in vscode with the current setup? Just wondering if we need to add a dependency on prettier from the root package.json + add the format-on-save config to the repo root vscode settings.

setting the format-on-save on the repo root works. thanks for pointing that out 😄

**Pull request merged**